### DISCUSSION TITLE
Maintainer PSA: Cross Repo Issues

### DISCUSSION THREAD

**Discussion by port19x** [pystardust/ani-cli#1047] [Ideas] [answered]
We can now, thanks to the latest [github update](https://github.blog/changelog/2023-02-23-github-issues-projects-february-23rd-update/), easily mention issues outside of our own repos.

To demonstrate:
One of the prime examples of kovidgoyal being a rude, out of touch oss dev can be found in kovidgoyal/kitty#4965

This renders `kovidgoyal/kitty#4965` as a link to that issue :)

For us this is mainly useful for the occasional feature request or issue related to mpv, ffmpeg or vlc. As well as to link to other affected projects during outages in resolving PRs

**Comment by Derisis13** [comment, marked as answer]
Cool.

I have spoken.
